### DISCUSSION TITLE
Move splits?

### DISCUSSION THREAD

**Discussion by e-minguez** [ghostty-org/ghostty#3325] [Feature Requests, Ideas] [answered]
iterm2 supports moving splits by dragging the cursor from the top (header?) of the split. Is it planned to have such (or similar) feature on ghostty?
Thanks!

**Comment by LogFlames** [comment]
I would love to be able to move splits as well.

Maybe something similar to kitty's [move_window](https://sw.kovidgoyal.net/kitty/layouts/) action.

**Reply by Xophmeister** [reply]
A `move_split` action would be great, taking the same direction arguments as `goto_split` (although, see https://github.com/ghostty-org/ghostty/discussions/2986 and https://github.com/ghostty-org/ghostty/discussions/3604), which would effectively swap the two splits. Personally, such a keybinding+action would be preferable to a mouse-based interaction.

**Comment by SenseiMarv** [comment]
A possible solution how this could be solved would be to introduce bars for splits as iTerm2 does: https://github.com/ghostty-org/ghostty/discussions/3747

**Comment by 00-kat** [comment, marked as answer]
Duplicate of #1525.

**Reply by bfontaine** [reply]
No it’s not because #1525 is only about macOS.

**Reply by bfontaine** [reply]
For Linux/GTK, see https://github.com/ghostty-org/ghostty/issues/10224
